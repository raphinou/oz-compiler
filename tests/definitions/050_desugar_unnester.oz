% Simple desugar of + - * / and unnesting of that operation's result assignment
local
   A = 8
   B = 2
   Add Sub Mult Div
in
   Add = A + B
   Sub = A - B
   Mult= A * B
   Div = A div B
   {Show Add}
   {Show Sub}
   {Show Mult}
   {Show Div}
end
