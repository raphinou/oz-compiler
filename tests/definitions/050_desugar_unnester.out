10
6
16
4
